lib/model.js

```diff
@@ -4556,7 +4556,7 @@ Model.validate = function validate(obj, pathsToValidate, context, callback) {
  * - match: optional query conditions to match
  * - model: optional name of the model to use for population
  * - options: optional query options like sort, limit, etc
- * - justOne: optional boolean, if true Mongoose will always set `path` to an array. Inferred from schema by default.
+ * - justOne: optional boolean, if true Mongoose will always set `path` to a document or `null`. If false, Mongoose will always set `path` to a potentially empty array. Inferred from schema by default.
```

Comment:
`"to a potentially empty array"` makes me question what that means. Is it a document/null if it's an id, null if not found., or an array (empty if none found) of documents if the path is an array of ids?

If my understanding is correct, I think we should be a bit more verbose in favor of explaining in more detail how it works.